web/src/components/tasks/LicenseTask.tsx

```diff
@@ -46,20 +45,23 @@ export const LicenseTask = (props: Props): ReactElement => {
 
   const licenseNameForTask = taskIdLicenseNameMapping[props.task.id];
   const hasCompletedSearch = !!licenseDetails?.lastUpdatedISO;
-  const searchHasError = !!licenseDetails?.hasError;
 
   const allFieldsHaveValues = (nameAndAddress: LicenseSearchNameAndAddress): boolean => {
     return !!(nameAndAddress.name && nameAndAddress.addressLine1 && nameAndAddress.zipCode);
   };
 
   useEffect(() => {
     if (!business) return;
-    const licenseDetailsReceived = business.licenseData?.licenses?.[licenseNameForTask]?.lastUpdatedISO;
+    const hasDoneLicenseCheck = business.licenseData?.lastUpdatedISO !== undefined;
```

Comment:
take a look at the initial of license data, it's initially set to undefined, then when the search is done, it's replaced with a object